client/common/cmdline.c

```diff
@@ -410,11 +410,28 @@ static size_t print_description(const char* text, size_t start_offset, size_t cu
 	return current;
 }
 
-static void freerdp_client_print_command_line_args(const COMMAND_LINE_ARGUMENT_A* arg)
+static int sort_cmdline_args(const void* pva, const void* pvb)
```

Comment:
This function itself does not sort things, just compare (as cmp func for the sorting):
```suggestion
static int cmp_cmdline_args(const void* pva, const void* pvb)
```

Reply:
ok, updated.